### ISSUE TITLE
Compatibility matrix for borker versions

### ISSUE THREAD

**Issue by hecko** [centreon/centreon-broker#727]
It would be extremely helpful to see a compatibility matrix between the broker server (cbd) and broker modules on pollers. 
Something that @julienmathis referred to in https://github.com/centreon/centreon-broker/issues/81#issuecomment-269475158 but I was unable to find such compatibility matrix on a website for recent versions of Centreon (20.10 and up).

Having a compatibility between different broker versions from different centreon distributions (pollers) is critical in designing a upgrade path between Centreon version especially when running multiple (hundreds) of pollers with one HA central. Its usually unfeasible to do a big-bang upgrade so the compatibility matrix is the only solution for this IMO. 

From tests I know that 21.04 poller module is compatible with 20.10 cbd, but I would love to see the full matrix.

Is such compatibility matrix available @sc979 ?

**Comment by sc979** [comment]
Hi @hecko ,
As you can see in my commits, I am not working on the development of this project.
Maybe @bouda1 will be able to help you.
Best regards,
sc979

**Comment by hecko** [comment]
RIght, thanks for correcting me. @bouda1 is there any chance you can compile compatibility matrix?
